charts/backbeat/values.yaml

```diff
@@ -1,6 +1,14 @@
 # Default values for backbeat.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+metrics:
+  enabled: true
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "8000"
```

Comment:
Does Prometheus support named ports in annotations? If so, that'd be a better approach.

Reply:
Looking at many other charts, it doesn't seem that it supports named ports.